src/services/MainWorker.ts

```diff
@@ -50,6 +50,14 @@ const onQueryResults = (queryProgress: number, queryResults: QueryResults) => {
     postResp(WORKER_RESP_CODE.QUERY_RESULT, {progress: queryProgress, results: queryResults});
 };
 
+/**
+ * Sends a message to the renderer to open a pop-up which prompts user to replace the default
+ * format string.
+ */
+const postFormatPopup = () => {
```

Comment:
For now this is fine. In a future PR, we can refactor notification & exception handling in the service worker so that every notification at any level can only be sent by either a notification dispatcher (like this `postFormatPopup`) or the exception catching ~L147. What do you think?

Reply:
Sure. Let's add the issue.